workflows/stairway.py

```diff
@@ -57,21 +57,36 @@ def ts_to_stairway(self, ts_path, num_bootstraps=1, mask_file=None):
             total_length += ts.sequence_length
             num_samples = ts.num_samples
             haps = ts.genotype_matrix()
-            # print(haps.shape)
+
+            # Mapping mutation type IDs to class of mutation (e.g., neutral, non-neutral)
+            mut_types = {}
+            for dfe in ts.metadata["stdpopsim"]["DFEs"]:
+                for mt in dfe["mutation_types"]:
+                    mid = mt["slim_mutation_type_id"]
+                    if not mid in mut_types:
+                        mut_types[mid] = "neutral" if mt["is_neutral"] else "non_neutral"
+
+            site_class = np.empty(ts.num_sites, dtype=object)
+
+            # Finding neutral positions
+            neu_positions = []
+            for j, s in enumerate(ts.sites()):
+                mt = []
+                for m in s.mutations:
+                    for md in m.metadata["mutation_list"]:
+                        mt.append(md["mutation_type"])
+                        if mut_types[md["mutation_type"]] == "neutral":
+                            neu_positions.append(m.site)
```

Comment:
This will append a site to `neu_positions` multiple times if it has multiple neutral mutations. Is this desired?

Reply:
Ah, I see that you do `set( )` down below to get rid of the duplications. How about instead
```
                    for md in m.metadata["mutation_list"]:
                        mt.append(md["mutation_type"])
                    if set(mt) == set("neutral"):
                         neu_positions.append(m.site)
```